metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/service/OrchestratorService.java

```diff
@@ -560,10 +564,12 @@ public DatasetExecutionInformation getDatasetExecutionInformation(MetisUser meti
         .getLatestSuccessfulExecutablePlugin(datasetId, HARVEST_TYPES, false).getPlugin();
```

Comment:
You should probably test this occurrence of the same method call as well!

Reply:
good point!